www/src/components/LeaveFeedback.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Nav } from '~paragon-react';
+import { useLocation } from '@reach/router';
 
 function LeaveFeedback(props) {
-  const CURRENT_URL = typeof window !== 'undefined' ? window.location.pathname : '';
-  const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${CURRENT_URL})&amp;labels=docs&template=feedback_template.md`;
+  const CURRENT_URL = useLocation();
+  const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${CURRENT_URL.pathname})&amp;labels=docs&template=feedback_template.md`;
```

Comment:
Maybe it is good idea to put it outside the component body?

Reply:
Thanks for the proposal!
The value of the `CURRENT_URL` variable is React Hook. React Hooks must be called in a React function component or a custom React Hook function.